### PR TITLE
Use `spawn` for multiple agents

### PR DESCRIPTION
**Pull request by DrJosh9000**
... on Debian, Red Hat, and Ubuntu docs.

### CODE REVIEW

File: pages/agent/v3/_linux_multiple_agents.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+You can run as many parallel agent workers on the one machine as you wish with
+the `spawn` configuration setting, or by passing the `--spawn` flag.
+
+```ini
+# Start 5 workers. Each one will independently fetch and execute jobs.
```
Comment (mbelton-buildkite) [inline pages/agent/v3/_linux_multiple_agents.md:5]: Style nitpick:

```suggestion
# Start 5 workers. Each one independently fetches and executes jobs.
```

File: pages/agent/v3/_linux_multiple_agents.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+spawn=5
+```
```
Comment (mbelton-buildkite) [inline pages/agent/v3/_linux_multiple_agents.md:7]: Ending the file with a blank line should fix the linting error:

````suggestion
```

````

**Review by mbelton-buildkite — APPROVED** [review]
Nice! Just a couple of small things to update

**Comment by DrJosh9000** [comment]
I'm wondering whether to keep the old text, or at least allude to using the systemd template as an alternative, since we still ship it.

**Comment by mbelton-buildkite** [comment]
> I'm wondering whether to keep the old text, or at least allude to using the systemd template as an alternative, since we still ship it.

Riiiight, so the existing text is still technically correct, it's just not our recommended approach? If that's the case, do you have a sense of how important it is to know the other method? Will anyone not be able to use the new approach you're documenting?

If it's needed, we can document it alongside this but make sure to emphasize which is the preferred method. If it's an edge case that shouldn't be needed, I'd rather omit it to keep things simple. I'm happy to talk it through if that's easier.

**Comment by DrJosh9000** [comment]
That helps, thanks @mbelton-buildkite. The `--spawn`-based setup is more generally compatible and anyone using a systemd unit template should be able to use `--spawn`. I see the main downside to deleting the old method might be that anyone who set it up that way might want to refer to some documentation to figure out what they did.

File: pages/agent/v3/_linux_multiple_agents.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+You can run as many parallel agent workers on the one machine as you wish with
+the `spawn` configuration setting, or by passing the `--spawn` flag.
+
+```ini
```
Comment (triarius) [inline pages/agent/v3/_linux_multiple_agents.md:4]: If only this were true 😂

**Review by triarius — APPROVED** [review]
Nice avoidance of duplication

File: pages/agent/v3/_linux_multiple_agents.md (hunk 3 above)
Comment (DrJosh9000) [reply pages/agent/v3/_linux_multiple_agents.md:4]: Details, details...

**Pull request merged**